inbm/Changelog.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Fixed
  - HSD 15010509095 - INBC fail to return correct Exit Code for difference scenario (-4/-6/-11)
+ - HSD 15010982715 - inbm-vision failed to receive query request from cloud
```

Comment:
this should not be in inbm changelog

Reply:
@gblewis1 - The reason we put it in there also is because the change is in the dispatcher-class file which affects INBM.